test/battle/ability/magician.c

```diff
@@ -0,0 +1,30 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Magician does not get self-damage recoil after stealing Life Orb")
+{
+    GIVEN {
+        ASSUME(gItems[ITEM_LIFE_ORB].holdEffect == HOLD_EFFECT_LIFE_ORB);
+        ASSUME(gBattleMoves[MOVE_TACKLE].power != 0);
+        ASSUME(P_GEN_6_POKEMON == TRUE);
```

Comment:
Edu changed it so that pokemon gen assumes are no longer needed. Maybe they can clarify on how it works.

Reply:
`PLAYER` and `OPPONENT` essentially check for `IsSpeciesEnabled`. If it returns FALSE, the test automatically ends as `ASSUMPTION_FAIL` with a message that says that the species was disabled.

Reply:
That's a great change! I removed the assume